pkg/expression/fulltext/query_test.go

```
@@ -0,0 +1,435 @@
+		ParserType:           model.FullTextParserTypeNgramV1,
+		InnodbFtMinTokenSize: 3,
+		InnodbFtMaxTokenSize: 84,
+		NgramTokenSize:       2,
+	}
+}
+
+// TestCompileBooleanQueryMultiTokenTerm covers a boolean term the analyzer
+// splits into several words, such as `foo.bar`. Each word is required: a
+// document matching the original term contains all of them. Previously such a
+// term was dropped, which made the whole query match no document at all.
+func TestCompileBooleanQueryMultiTokenTerm(t *testing.T) {
```

golangci-lint (2.12.2)

[warning] function-length: maximum number of statements per function exceeded; max 10 but got 22

(revive)